docs/UserGuide.md

```diff
@@ -691,6 +720,15 @@ _You can delete any customers who no longer require your services from HomeBoss
 
 **Format:** `customer delete CUSTOMER_ID`{.swift}
 
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* `CUSTOMER_ID`{.swfit} must be an integer greater than 0.
```

Comment:
`CUSTOMER_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Customer in the address book.